tests/bluetooth/tester/src/bttester.c

```diff
@@ -187,7 +196,7 @@ static u8_t *recv_cb(u8_t *buf, size_t *off)
 		return buf;
 	}
 
-	k_fifo_put(&cmds_queue, buf);
+	k_fifo_put(&cmds_queue, CONTAINER_OF(buf, struct btp_buf, data));
 
 	*off = 0;
 	return new_buf;
```

Comment:
Shouldn't this be cast to 'struct btp_buf' and then return buf->data. Now it seems you're returning a pointer to the new _reserved member.

Reply:
Done.